exercises/hamming/src/Hamming.idr

```diff
@@ -0,0 +1,10 @@
+module Hamming
+
+import Data.Vect
+
+-- Add the correct export modifier here...
+data Nucleotide = A | C | G | T
```

Comment:
Idris doesn't permit a hole for a constructor. 
What about an empty data type + comment:
```data Nucleotide -- insert definition here```